src/blocks/notifications/Notifications.tsx

```diff
@@ -0,0 +1,149 @@
+    font-weight: ${textVariants['h5-semibold'].fontWeight};
+    line-height: ${textVariants['h5-semibold'].lineHeight};
+  }
+
+  .description {
+    color: var(--components-in-app-notification-text-secondary);
+    font-family: var(--font-family);
+    font-size: ${textVariants['bes-regular'].fontSize};
+    font-style: ${textVariants['bes-regular'].fontStyle};
+    font-weight: ${textVariants['bes-regular'].fontWeight};
+    line-height: ${textVariants['bes-regular'].lineHeight};
+  }
+`;
+
+const IconContainer = styled.div`
+  padding: var(--spacing-sm) var(--spacing-xs);
+  border-radius: var(--radius-xxs) var(--radius-none) var(--radius-none) var(--radius-xxs);
+  background: radial-gradient(79.55% 79.55% at 50% 50%, #344efd 0%, #171717 100%);
+`;
+
+const CloseButton = styled.div`
+  background-color: var(--surface-transparent);
+  cursor: pointer;
+  color: var(--components-in-app-notification-icon-default);
+  padding: var(--spacing-none);
+  position: absolute;
+  right: var(--spacing-xxs);
+  top: var(--spacing-xxs);
+`;
+
+const Notifications: FC<NotificationProps> = ({
+  isOpen,
+  onClose,
+  title,
+  description,
+  icon,
+  onClick,
+  duration = Infinity,
+  position = 'bottom-right',
+}) => {
+  const handleNotificationClick = () => {
+    if (onClick) onClick();
+    handleNotificationClose();
+  };
+
+  const handleNotificationClose = () => {
+    if (onClose) onClose();
+    toast.dismiss();
+  };
+
+  useEffect(() => {
+    if (isOpen) {
+      toast.custom(() => (
+        <NotificationContainer onClick={handleNotificationClick}>
+          <IconContainer>{icon}</IconContainer>
+          <CloseButton
+            onClick={(e) => {
+              e.stopPropagation();
+              handleNotificationClose();
+            }}
+          >
+            <Cross size={16} />
+          </CloseButton>
+          <TextContainer>
+            <span className="title">{title}</span>
+            <span className="description">{description}</span>
+          </TextContainer>
+        </NotificationContainer>
+      ));
+    }
+  }, [isOpen]);
+
+  return (
+    <Toaster
+      style={{ width: '397px', height: '78px' }}
+      visibleToasts={1}
+      offset={15}
+      duration={duration || Infinity}
+      position={position}
+    />
+  );
+};
+
+const renderNotification = (props: NotificationProps) => {
+  const div = document.createElement('div');
+  document.body.appendChild(div);
+
+  const root = ReactDOM.createRoot(div);
+
+  const handleClose = () => {
+    root.unmount();
+    document.body.removeChild(div);
+    if (props.onClose) props.onClose();
+  };
+
+  root.render(
+    <Notifications
+      {...props}
+      onClose={handleClose}
+    />
+  );
+};
+
+const notifications = {
+  show: (config: Omit<NotificationProps, 'isOpen'>) => {
+    renderNotification({ ...config, isOpen: true });
+  },
+  hide: () => {
+    toast.dismiss();
+  },
+};
+
+export { Notifications, notifications };
```

Comment:
```suggestion
export {  notification };
```
We don't need to export the component.